---
bool success
sensor_msgs/Image img